projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -119,10 +119,16 @@ function process_anchor_params() {
 							),
 						);
 
+						$self_links = $rss->get_links( 'self' );
+						$cover      = $rss->get_image_url();
+						$cover      = ! empty( $cover ) ? esc_url( $cover ) : null;
```

Comment:
Maybe we can escape later?

```suggestion
						$cover      = ! empty( $rss->get_image_url() ) ? $rss->get_image_url() : null;
```

Reply:
Definitely! Is the advantage that we escape at the last minute, in case the value gets used in a future change?

Reply:
That would be the idea, yes.